CMakeLists.txt

```diff
@@ -37,6 +37,11 @@ endif()
 
 include(cmake/utils.cmake)
 
+if(NOT TANGRAM_CXX_STANDARD)
+    message(STATUS "CXX STANDARD NOT SET. Setting to c++14")
```

Comment:
Not sure this is necessary, wouldn't want to make someone think they've improperly configured the build because of a message like this.